readthedocs/rtd_tests/tests/test_project_forms.py

```diff
@@ -29,6 +30,7 @@
     EmailHookForm
 )
 from readthedocs.projects.models import EnvironmentVariable, Project
+from readthedocs.projects.tasks import rename_project_dir
```

Comment:
I have implemented this task in PR #4995 ([code](https://github.com/rtfd/readthedocs.org/pull/4995/files#diff-b9399e1d3499066c5564f98a620e8881R1458)) which is yet to be merged. It is better to merge this PR after that for the reusability of the function.

Reply:
Also, Travis will fail for now.